iquip/apps/scheduler.py

```diff
@@ -115,15 +109,14 @@ def __init__(self, info: ExperimentInfo, parent: Optional[QWidget] = None):
             info: The information of the experiment.
         """
         super().__init__(parent=parent)
-        self.experimentInfo = info
         # widgets
-        self.nameLabel = QLabel(info.name, self)
+        nameLabel = QLabel(info.name, self)
```

Comment:
Well, public attributes are not bad! We usually set the widgets as public attributes.
The main layout, however, can be accessed with `layout()` method which is defined in `QWidget`, so there is no need to set it a public attribute.